client/lib/plans/constants.js

```diff
@@ -591,6 +654,51 @@ export const PLANS_LIST = {
 		getPathSlug: () => 'business-2-years',
 	},
 
+	[ PLAN_ECOMMERCE ]: {
+		...getPlanEcommerceDetails(),
+		term: TERM_ANNUALLY,
+		getBillingTimeFrame: WPComGetBillingTimeframe,
+		availableFor: plan =>
+			includes(
```

Comment:
I think for the soft launch we'll want to remove upgrades. We'll be able to add upgrades after Atomic V2 is ready to accept transferring existing sites

Reply:
I have removed these plans in 1378518, thanks for the clarification!